File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.js
function solution(numbers) {
    var answer = [];
    
    for(let i = 0; i< numbers.length ; i++){
        for(let j = i + 1 ; j < numbers.length ; j++){
            let e = numbers[i] + numbers[j]
            if(!answer.includes(e)){
                answer.push(e)
            }
        }
    }
    
    answer.sort((a, b) => a - b)
    
    return answer;
}

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] numbers) {
        
        
        TreeSet<Integer> set = new TreeSet<>();
        
        for(int i = 0; i < numbers.length; i++){
            for(int j = i+1; j < numbers.length; j++){
                set.add(numbers[i] + numbers[j]);        
            }
        }
        int[] answer = new int[set.size()];
        for(int i = 0; i < answer.length; i++){
            answer[i] = set.pollFirst();
        }
        
        return answer;
    }
}